components/LotSubmissionForm.js

```diff
@@ -22,10 +22,10 @@ export default class LotSubmissionForm extends Component {
 			fullAddress: 'Search',
 		},
 		dropoffLocation: {},
-		offer: 0,
+		offer: 20,
```

Comment:
did u forget this here or why is offer 20 and pickupTime 15?

Reply:
I just though it seemed nicer to have the default values be more realistic, rather than 0. I think the 15 minutes is fine, but the offer being a value in the beginning might confuse people?

Reply:
Yeah the minutes make sense, not quite sure about the offer